stdlib/transactions/src/main/ballerina/transactions/commons.bal

```diff
@@ -164,6 +164,11 @@ function getParticipantProtocolAt(string protocolName, int transactionBlockId) r
 
 # The initiator will create a new transaction context by calling this function. At this point, a transaction object
 # corresponding to the coordinationType will also be created and stored as an initiated transaction.
+#
+# + coordinationType - The type of the coordination relevant to the transaction block for which this TransactionContext
+#                      is being created for
+# + transactionBlockId - The ID of the transaction block
+# + return - The created TransactionContext or error if the coordinationType is invalid
```

Comment:
Shall we reword this to "TransactionContext if the coordination type is valid or an error in case of an invalid coordination type."